src/Sql/Sql.Test/ScenarioTests/FailoverGroupTests.cs

```diff
@@ -84,6 +84,13 @@ public void TestCreateFailoverGroup_Overflow()
             RunPowerShellTest("Test-CreateFailoverGroup-Overflow");
         }
 
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestCreateFailoverGroup_CrossSubscription()
+        {
+            RunPowerShellTest("Test-CreateFailoverGroup-CrossSubscription");
```

Comment:
As our test framework is adjusted, please pull latest main into your branch and change this Line to 
```suggestion
            TestRunner.RunTestScript("Test-CreateFailoverGroup-CrossSubscription");
```

Reply:
Pulled in latest changes from Azure/azure-powershell:main into SharanSingh/azure-powershell:main and updated the test method name from `RunPowerShellTest` to `TestRunner.RunTestScript`